gui-js/libs/ui-components/src/lib/import-csv/import-csv.component.ts

```diff
@@ -195,7 +200,8 @@ export class ImportCsvComponent extends Zoomable implements OnInit, AfterViewIni
       this.selectRowAndCol(this.dialogState.spec.dataRowOffset, this.dialogState.spec.dataColOffset);
     })();
   }
-  ngAfterViewChecked() {
+
+  afterViewChecked() {
```

Comment:
What went wrong here?

Reply:
I'm reverting this change, but I am adding the AfterViewChecked interface to make clearer this is referring to a component lifecycle hook (did it even work without that interface..?)

Reply:
I guess using the interface is optional, but strongly recommended practice in Angular.

Reply:
The comment that calling an ng-prefixed method from own code is bad practice also applies here... it's what confused me to make the initial change here. Those "checked" hooks are a bit of a can of worms by the way. If the change detection strategy of the component is not set to "On Push"*, the onChecked hook methods get called extremely often, on things like mouse move events and in response to anything changing in the component template... These methods should never contain performance intensive logic and their use should preferably be avoided.

* at my current company we have a policy to use this change detection strategy on most of our components... refactoring our code to this (which mostly prevents negative performance surprises) would require changing a lot of variables that are dynamically read from the HTML template to rxjs Subjects with .next used to set their value. Probably a bridge too far at this point, but worth knowing about.